app/src/main/kotlin/com/wire/android/ui/authentication/create/username/CreateAccountUsernameViewModel.kt

```diff
@@ -59,4 +60,11 @@ class CreateAccountUsernameViewModel @Inject constructor(
                 navigationManager.navigate(NavigationCommand(NavigationItem.Home.getRouteWithArgs(), BackStackMode.CLEAR_WHOLE))
         }
     }
+
+    companion object {
```

Comment:
how about moving this to some common packagage so we can reuse it if needed ?

Reply:
Yep, I created UsernameUtil.